salt/cloud/clouds/vmware.py

```diff
@@ -1872,3 +1872,62 @@ def upgrade_tools(name, reboot=False, call=None):
         return 'VMware tools upgraded'
 
     return 'VMware tools is not installed'
+
+def rescan_all_hbas_on_host(kwargs=None, call=None):
+    '''
+    To rescan all HBA's on a Host using the Hostname
+        
+    CLI Example:
+        
+    .. code-block:: bash
+        
+    salt-cloud -f rescan_all_hbas_on_host my-vmware-config host
+    '''
+    
+    if call != 'function':
+        raise SaltCloudSystemExit(
+            'The rescan_all_hbas_on_host action must be called with -f or --function.'
```

Comment:
``The rescan_all_hbas_on_host function`` instead of ``The rescan_all_hbas_on_host action``